resources/markdown/samReadme.md

```diff
@@ -0,0 +1,32 @@
+# Developing AWS SAM Applications with the ${PRODUCTNAME}
+
+This project contains source code and supporting files for a serverless application that you can locally run, debug, and deploy to AWS with the ${PRODUCTNAME}.
+
+## Writing and Debugging Serverless Applications
```

Comment:
We should include some information somewhere about setting up your toolchain to be able to debug. Maybe link directly to `https://docs.aws.amazon.com/toolkit-for-vscode/latest/userguide/setup-toolchain.html` (could be in the resources section)?

Reply:
Good thought. The difficult part is sliding this into Cloud9 as well since there's no toolchain required for that.